src/Controller.h

```diff
@@ -70,6 +70,7 @@ class Controller
   void indicateActivity();
   void setLearnMode(byte reqMode);
   void setFrameHandler(void (*fptr)(CANFrame *msg), byte *opcodes = NULL, byte num_opcodes = 0);
+  byte newMode;
```

Comment:
This variable can be placed in the private part. EventProducerService is a "friend" which means it has access to everything in Controller.

Reply:
I did wonder.  Now moved.